src/main/webapp/intro.html

```diff
@@ -31,29 +36,29 @@ <h5 class="section-title h1">Team NAN</h5>
                 <div class="card-body text-center">
                   <p><img class=" img-fluid" src="images/Molly.jpg" alt="card image"></p>
                   <h4 class="card-title">Molly Graton</h4>
-                  <p class="card-text">Descibe Molly in a sentence or less. IAmAPlaceholder IAmAPlaceholder.</p>
+                  <p class="card-text">Google STEP Intern</p>
                 </div>
               </div>
             </div>
             <div class="backside">
               <div class="card">
                 <div class="card-body text-center mt-4">
-                  <h4 class="card-title">Molly</h4>
+                  <h4 class="card-title">Molly Graton</h4>
                   <p class="card-text">Molly is a rising sophomore at University of Illinois Urbana-Champaign studying computer science. </p>
                   <ul class="list-inline">
                     <li class="list-inline-item">
                       <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
-                        <i class="fa fa-google"></i>
+                        <i class="fa fa-linkedin"></i>
                       </a>
                     </li>
                     <li class="list-inline-item">
-                      <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
-                        <i class="fa fa-facebook"></i>
+                      <a class="social-icon text-xs-center" target="_blank" href="https://github.com/mollygraton">
+                        <i class="fa fa-github"></i>
                       </a>
                     </li>
                     <li class="list-inline-item">
-                      <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
-                        <i class="fa fa-twitter"></i>
+                      <a class="social-icon text-xs-center" target="_blank" href="molly.graton@gmail.com">
```

Comment:
You probably should update these email links to "mailto:" links

I think something like this should work:

`mailto:molly.graton@gmail.com?subject=Extremely Important SAY Inquiry`